packages/esm-active-visits-app/src/active-visits-widget/active-visits.component.tsx

```diff
@@ -91,7 +97,11 @@ const ActiveVisitsTable = (props) => {
       return activeVisits;
     }
   }, [searchString, activeVisits]);
-  const { goTo, currentPage, results } = usePagination(searchResults, currentPageSize);
+  const { goTo, results } = usePagination(searchResults, currentPageSize);
+
+  useEffect(() => {
+    goTo(currentPage);
```

Comment:
I don't get this side-effect. It will essentially always trigger...

Reply:
Actually the memo I used above, it's triggered whenever I am searching something, I want to set page no. back to 1, but `goTo` isn't initialized when memo is being used, hence I use the `setCurrentPage` to set page back to 1 and when `currentPage` value changes, this function is triggered. It will not be triggered always, because `currentPage` is used only when searching.

Reply:
Will look for work around

Reply:
Your logic does not make much sense. Please read carefully what I wrote. The effect is used initially, which triggers `goTo` potentially changing `currentPage`. Let's say it did not change (fine), then the effect is not applied. But let's say it did change, then `currentPage` is changed (wtf? this case cannot even be unless you made a lot of boo-boos.).

This side-effect and the currentPage in the useMemo are both not needed at all if you model that thing correctly.

Reply:
Sure @FlorianRappl I got your point, just tried to share what was on my mind the first time, can you review the new changes I made, maybe they might be good to go.

Reply:
Looks good!